manim/mobject/svg/svg_path.py

```diff
@@ -217,7 +217,14 @@ def grouped(iterable, n):
 
 
 class SVGPathMobject(VMobject, metaclass=ConvertToOpenGL):
-    def __init__(self, path_string, **kwargs):
+    def __init__(
+        self,
+        path_string,
+        should_subdivide_sharp_curves=False,
+        should_remove_null_curves=False,
```

Comment:
I think these should be removed and just only passed by `SVGMobject` if the renderer is opengl, particularly since now these are just being consumed without being forwarded on